src/serac/numerics/mesh_utils.hpp

```diff
@@ -110,12 +110,14 @@ std::shared_ptr<mfem::ParMesh> buildCylinderMesh(int radial_refinement, int elem
  * @param[in] inner inner radius the radius of the cylindrical shell
  * @param[in] outer ouer radius the radius of the cylindrical shell
  * @param[in] height the number of elements in the z-direction
+ * @param[in] sectors the number of starting sectors in the hollow cylinder
```

Comment:
what is "angle"? angle per sector, total angle?
I'd prefer specifying a total angle, personally.

Reply:
Hmm. I think this implementation assumes that the elements need to be integer multiples of the "sector angle". I agree that total angle is more convenient from a specification standpoint, but not sure what the best way to check the required condition (floating point considerations).

Reply:
hmm, I don't understand, isn't the sector angle just total_angle / num_angular_divisions?

Reply:
one option is to make total_angle the last argument, and default it to 2 * pi. The body of the function can do something like
```
if (min(total_angle, 2 * M_PI) == 2 * M_PI) {
  // full cylinder
} else {
  // partial cylinder
}
```

Reply:
Sorry, I think I understand what you are suggesting now. Specify `total_angle` and `num_angular_sectors` instead of `sector_angle` and `num_angular_sectors`, correct? I think that's a good idea.